qqmlsortfilterproxymodel_plugin.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <QQmlExtensionPlugin>
+
+class QQmlsortfilterproxymodelPlugin : public QQmlExtensionPlugin
```

Comment:
I assume we'll also need `QQMLSORTFILTERPROXYMODEL_EXPORT` here, but @crozeg's Windows tests should be able to confirm it.